src/components/dropdown-menu/ResponsiveMenuContainer.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../styles/variables';
+
+.menu-container {
+    position: relative;
+    top: 0;
+    left: 0;
+    z-index: 9999;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    background-color: $bdl-gray-02;
+
+    .menu-header {
+        position: relative;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 13px 16px;
```

Comment:
Can we investigate if there are any existing variables we can use for padding, box shadow, font-size, height, width, etc.?